libs/deno/runtime.js

```diff
@@ -0,0 +1,11 @@
+// console.log("runtime.js is running!");
+
+// const { core } = Deno[Deno.internal];
+const { core } = Deno;
```

Comment:
no ts supported (I saw they are ignored?)

Reply:
I don't think so, they're using JS in the deno repo as well. I'll look into it, we can probably add a buildstep if necessary.